docs/server/push-duty.rst

```diff
@@ -99,10 +98,32 @@ Then update the etherpad with the new comparison link for the updated tag.
 Push to stage
 +++++++++++++
 
-Once the tag is ready and any needed docker containers have been built, it
+Once the tag is ready and any needed
+`Docker Hub`_ containers have been built, it
 needs to be pushed to the staging server, using the `IRC bot in #amopush
 <https://mana.mozilla.org/wiki/display/SVCOPS/AMO+Dev+Resources#AMODevResources-DeployingwithJenkinsBot>`_.
-Jason can help with that if needed.
+Consult the IRC bot docs for more up to date information but here are examples
```

Comment:
Well, it's hard to read ReST/Sphinx diffs. I think it should be clear in the HTML.